client/src/app/admin/profile/profile.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, OnInit } from '@angular/core';
+import { UserService } from 'src/app/core/services/user/user.service';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { MatSnackBar } from '@angular/material';
+
+@Component({
+  selector: 'em-profile',
+  templateUrl: './profile.component.html',
+  styleUrls: ['./profile.component.scss']
+})
+export class ProfileComponent implements OnInit {
+
+  constructor(private userService: UserService, private snackBar: MatSnackBar) { }
+  user: any;
+  editSection = false ;
+  EditInfoGroup: FormGroup;
```

Comment:
Rename `EditInfoGroup` to `editInfoForm`.
Always use `camelCase` for naming properties.

Reply:
done